tests/app/submitter/test_converter.py

```diff
@@ -93,24 +68,54 @@ def test_prepare_answers(self):
             questionnaire.register(answer_1)
             questionnaire.register(answer_2)
 
-            answer_object, submitted_at = Converter.prepare_answers(metadata, questionnaire, user_answer)
+            answer_object, _ = convert_answers(metadata, questionnaire, AnswerStore(user_answer), {})
 
-            self.assertEquals(EXPECTED_RESPONSE["type"], answer_object["type"])
-            self.assertEquals(EXPECTED_RESPONSE["version"], answer_object["version"])
-            self.assertEquals(EXPECTED_RESPONSE["origin"], answer_object["origin"])
-            self.assertEquals(EXPECTED_RESPONSE["survey_id"], answer_object["survey_id"])
-            self.assertEquals(EXPECTED_RESPONSE["collection"], answer_object["collection"])
-            self.assertEquals(EXPECTED_RESPONSE["metadata"], answer_object["metadata"])
-            self.assertEquals(EXPECTED_RESPONSE["paradata"], answer_object["paradata"])
-            self.assertEquals(EXPECTED_RESPONSE["data"], answer_object["data"])
+            assert_that(answer_object).has_type("uk.gov.ons.edc.eq:surveyresponse") \
```

Comment:
I have used a assertion libary called [assertpy](https://github.com/ActivisionGameScience/assertpy) here. It's worth taking a look at the API and get a general feeling of if it's worth using as it has some good utilities and a fluent DSL